services/drupal/web/modules/custom/epa_workflow/src/EPAModeration.php

```diff
@@ -223,6 +223,13 @@ protected function setReviewDeadline($reset = FALSE) {
       $date->add(new \DateInterval("P{$review_period}D"));
       $review_deadline = $date->format(DateTimeItemInterface::DATETIME_STORAGE_FORMAT);
       $this->contentEntityRevision->set('field_review_deadline', $review_deadline);
+      if ($bundle == 'public_notice' && !$this->contentEntityRevision->get('field_notice_sort_date')->isEmpty()) {
+        $sort_date = new DrupalDateTime($this->contentEntityRevision->field_notice_sort_date->value);
```

Comment:
I'm thinking there should be a way to get a DrupalDateTime object out of the field directly rather than creating a new one here and having to set the timezone...but I could be wrong.  Could you maybe look into this?